style.css

```diff
@@ -437,3 +455,125 @@ h2.section-title {
   height: 100%;
   z-index: -111;
 }
+
+@media (min-width: 786px) {
+  .container {
+    max-width: 1156px;
+    margin: 0 auto;
+  }
+
+  .nav-links {
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+  }
+
+  .nav-links li {
+    display: flex;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    padding-right: 20px;
+    color: #344563;
+  }
+
+  i.fas.fa-bars {
+    display: none;
+  }
+
+  .work-description {
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+  }
+
+  .headline-wrap {
+    background-image: url(image/back-desktop.svg);
+    background-position: center;
+  }
+
+  .greetings {
+    font-weight: 700;
+    font-size: 48px;
+    max-width: 500px;
+  }
+
+  .headline-profile {
+    max-width: 920px;
+    font-size: 20px;
+  }
+
+  .works-grid-item {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 20px;
+    align-items: center;
+  }
+
+  .card-content {
+    max-width: 496px;
+  }
+
+  .works-grid-item img {
+    max-width: 544px;
+  }
+
+  .works-grid-item:nth-child(2) img {
+    grid-column: 2/2;
+    grid-row: 1;
+  }
+
+  .works-grid-item:nth-child(2) .card-content {
+    grid-column: 1/2;
+  }
+
+  .works-grid-item:nth-child(4) img {
+    grid-column: 2/2;
+    grid-row: 1;
+  }
+
+  .works-grid-item:nth-child(4) .card-content {
+    grid-column: 1/2;
+  }
+
+  .about-section {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    align-items: center;
+    gap: 10px;
+  }
+
+  .content-aboutleft {
+    max-width: 447px;
+  }
+
+  .content-aboutleft p {
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+  }
+
+  .other-info-details {
+    display: flex;
+    align-items: center;
+    gap: 10px;
+  }
+
+  .view-list-item {
+    display: block;
+  }
+
```

Comment:
- [ ] Please head to your styles where you declared CSS rules for the contact-form section. I would like to talk about the paragraph with `class='Test-description'`. It would be good if you give it a `max-width` so that **It does not take up all the space when the window screen size increase**. Take a look at the comparison below 👇 

**Your Design**
![image](https://user-images.githubusercontent.com/90222110/186703968-8748c89f-521a-4069-9bfc-6cb153721afb.png)

**Figma**
![image](https://user-images.githubusercontent.com/90222110/186704155-bb568d16-4023-4d58-b9af-3a85ce18bd5c.png)